bridge-token-factory/src/lib.rs

```diff
@@ -61,6 +61,9 @@ const METADATA_CONNECTOR_ETH_ADDRESS_STORAGE_KEY: &[u8] = b"aM_CONNECTOR";
 /// The prefix is made specially short since it becomes more expensive with larger prefixes.
 const TOKEN_TIMESTAMP_MAP_PREFIX: &[u8] = b"aTT";
 
+const FEE_DECIMAL_PRECISION: u128 = 1000000;
+const AURORA_ID: &str = "aurora";
```

Comment:
This shouldn't be fixed, as this could be an Aurora silo.

Reply:
Can you please guide me, what predecessor account id should be when withdrawing from AURORA?

Reply:
It should be flexible - it should be allowed to be configurable. Also, maybe it makes sense to allow subsets of accounts, e.g., the admin configures the address of Aurora as `aurora` and sets the flag to allow subset silos. This will mean that all the accounts in the domain will be treated as Aurora accounts, e.g., `silo1.aurora`, `some_other_silo.aurora`, etc. However, this will definitely need to have an exception for Aurora XCC accounts on NEAR (the ones containing in the domain name an EVM address - 20bytes + `.aurora`)

Reply:
added in 7684b501ede66a8c1e151cfc89af0d486310f25f

Reply:
It is still hardcoded, please make it configurable.

Reply:
Also, I would recommend holding a map of allowed engine accounts like in the [aurora-eth-connector](https://github.com/aurora-is-near/aurora-eth-connector/blob/master/eth-connector/src/lib.rs#L57)
`known_engine_accounts`

Reply:
yes. so I think that the `if withdraw_fee_percentage_per_silo.contain(withdrawer) {` is enough and better then `if is_aurora_engine_account(withdrawer.clone())`.
Actually, I think it can work as an exception list so that any account could be added to it, even some `DEX`

Reply:
cce1ee19211a8a00b1a8567e867eb183c5e24e98